LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 1998-2021 Brian Masney
```

Comment:
Per my comment below, you should also add a second line copyright line here for you as well and throughout if you desire.